### PR TITLE
Feature/update now config

### PR DESCRIPTION
**Pull request by RonanC**
Now didn't work for me like it did in your video.
I had to put the default react now.json file into the project, and update the package.json with two scripts.

### CODE REVIEW

File: catch-of-the-day/now.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "version": 2,
+  "name": "cotd",
+  "builds": [{ "src": "package.json", "use": "@now/static-build" }],
+  "routes": [
+    { "src": "^/static/(.*)", "dest": "/static/$1" },
```
Comment (ericrowan) [inline catch-of-the-day/now.json]: Looks great, Ronan. I believe we should add a route for images:
`{ "src": "^/images/(.*)", "dest": "/images/$1" },`

**Comment by dusterbed** [comment]
THANK YOU! This totally helped me with the zeit piece :) 💯

**Comment by RonanC** [comment]
@wesbos looks like others have come across this same issue, might be worth getting this merged?

**Comment by wesbos** [comment]
Merged! Thank you!

**Comment by RonanC** [comment]
@wesbos now problem, thanks for merging 😁

**Pull request merged**